whisperv6/peer.go

```diff
@@ -81,12 +82,16 @@ func (peer *Peer) handshake() error {
 	errc := make(chan error, 1)
 	isLightNode := peer.host.LightClientMode()
 	isRestrictedLightNodeConnection := peer.host.LightClientModeConnectionRestricted()
+	var rlCfg *ratelimiter.Config
```

Comment:
So this is effectively "this node ingress config"? Maybe we can be more explicit here?